config/routes.rb

```diff
@@ -5,6 +5,7 @@
   match '/services/*path', to: 'application#cors_preflight', via: [:options]
 
   get '/saml/metadata', to: 'saml#metadata'
+  get '/saml/metadata_v2', to: 'saml#metadata_v2'
```

Comment:
For routes, I was thinking clean up the paths somewhat to  align with everything under a v1/sessions_controller:

```
/v1/sessions/metadata
/v1/sessions/callback
/v1/sessions/logout
/v1/sessions/:type:/new
```